docs/UserGuide.md

```diff
@@ -135,7 +131,12 @@ Examples:
 * `find y/sec 3` matches `sec 3`, `Secondary 3`
 * `find y/sec 3` matches `sec 4`
 * `find n/alex s/yishun y/sec 3` searches for all students who match all of `n/alex`, `s/yishun` and `y/sec 3`.
-* 
+
+### Finding students with overdue fees: `unpaid`
```

Comment:
I think you might have to change this to overdue as well!

Reply:
Oh ya horh